ledger-core/virtual/execute/execute.go

```diff
@@ -737,6 +739,56 @@ func (s *SMExecute) stepSaveNewObject(ctx smachine.ExecutionContext) smachine.St
 	return ctx.Jump(s.stepSendCallResult)
 }
 
+func (s *SMExecute) stepPublishCallSummaryData(ctx smachine.ExecutionContext) smachine.StateUpdate {
+	syncAccessor, ok := callsummary.GetSummarySMSyncAccessor(ctx, s.execution.Object)
+
+	if ok {
+		var syncLink smachine.SyncLink
```

Comment:
I'd move this to separate step

Reply:
Fixed

Reply:
It's not pushed yet?

Reply:
Pushed

Reply:
stepAwaitSMCallSummary
stepPublishDataCallSummary